collectors/python.d.plugin/memcached/metadata.yaml

```diff
@@ -1,175 +1,247 @@
+# yamllint disable rule:line-length
+---
 meta:
   plugin_name: python.d.plugin
   module_name: memcached
   monitored_instance:
     name: Memcached
-    link: ''
+    link: https://memcached.org/
     categories:
-    - data-collection.database-servers
-    icon_filename: 'memcached.png'
+      - data-collection.database-servers
+    icon_filename: "memcached.png"
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
-    description: ''
-  keywords: []
+    description: ""
+  keywords:
+    - memcached
+    - memcache
+    - cache
+    - database
   most_popular: false
 overview:
   data_collection:
-    metrics_description: 'Monitor Memcached metrics for proficient in-memory key-value store operations. Track cache hits, misses, and memory usage for efficient data caching.'
-    method_description: ''
+    metrics_description: "Monitor Memcached metrics for proficient in-memory key-value store operations. Track cache hits, misses, and memory usage for efficient data caching."
+    method_description: ""
```

Comment:
Can you follow the style of https://github.com/netdata/netdata/pull/15528/files for example? method description can't be empty

Reply:
done here https://github.com/netdata/netdata/pull/15529/commits/79bb83ca0cbddd5c1911c5d078576e6de91b5eac